pkg/catalogsourceconfig/registry.go

```diff
@@ -369,3 +413,16 @@ func getRules(operatorSources []string) []rbac.PolicyRule {
 		NewRule([]string{"get"}, []string{""}, []string{"secrets"}, nil),
 	}
 }
+
+// hasCommandChanged returns true if the new command is different from the old
+func hasCommandChanged(old, new []string) bool {
+	if len(old) != len(new) {
```

Comment:
Couldn't this result in a false positive? For example if one operator was switched out with another of the same length?

Reply:
I am checking for `!=`. If I was using `==` and returning false then what you say is true.